var/spack/repos/builtin/packages/adios/package.py

```diff
@@ -50,11 +50,12 @@ class Adios(AutotoolsPackage):
 
     variant('mpi', default=True, description='Enable MPI support')
     variant('infiniband', default=False, description='Enable infiniband support')
+    variant('mxml', default=False, description='Use external MXML library')
 
     # transforms
     variant('zlib', default=True, description='Enable zlib transform support')
     variant('szip', default=False, description='Enable szip transform support')
-    variant('zfp', default=False, description='Enable ZFP transform support')
+    variant('zfp', default=False, description='Use external ZFP library')
```

Comment:
Sure, but without +zfp ADIOS will take its internal version anyhow. So even with default=True, the description is still "Use external ZFP" and not "enable/disable ZFP".
Similarly for MXML...

So then it is really just a ma